### PR TITLE
fix(INT-921): update Telegraf version to 1.17.2

### PR DESCRIPTION
**Pull request by svinothini24**
**PR description:**
This PR is regarding the replacement of cadvisor with the Telegraf agent to scrape the metrics from AWS ECS service.

Please find below the image available in the AWS ECR for testing purpose(The docker image is yet to be made publicly available).
358321641467.dkr.ecr.us-west-2.amazonaws.com/svinothini-wavefront:telegraf-ecs-test

For example,
"containerDefinitions": [
```
{
    "name": "aws-ecs-ec2-wavefront",
    "image": "358321641467.dkr.ecr.us-west-2.amazonaws.com/svinothini-wavefront:tele-exclude-labels",
    "memory": "150",
}
```

### CODE REVIEW

**Comment by svinothini24** [comment]
@srinivas-kandula @yogeshprasad @gangadharaswamy @skajagar Please review

File: aws-ecs/telegraf.conf [hunk 1]
```diff
@@ -54,7 +54,7 @@
 
   ## ecs labels to include and exclude as tags.  Globs accepted.
   ## Note that an empty array for both will include all labels as tags
-  ecs_label_include = [ "com.amazonaws.ecs.*" ]
```
Comment (srinivas-kandula) [inline aws-ecs/telegraf.conf:57]: How many labels are we getting by default and how many with this include tag?

Comment (svinothini24) [reply aws-ecs/telegraf.conf:57]: Below are the labels available at the filter
•	com.amazonaws.ecs.cluster
•	com.amazonaws.ecs.container-name
•	com.amazonaws.ecs.task-definition-family
•	com.amazonaws.ecs.task-arn
•	com.amazonaws.ecs.task-definition-version
•	cluster
•	family
•	id
•	name
•	network
•	revision
•	task_arn
•	source
•	tag

File: aws-ecs/telegraf.conf [hunk 2]
```diff
@@ -54,8 +54,8 @@
...
-  ecs_label_exclude = []
+  ecs_label_include = []
+  ecs_label_exclude = [ "com.amazonaws.ecs.*" ]
```
Comment (svinothini24) [inline aws-ecs/telegraf.conf:58]: This change excludes the point tags with prefix - "com.amazonaws.ecs.*" avoiding the duplicates

**Review by srinivas-kandula — APPROVED** [review]
LGTM

**Pull request merged**